deploy/crds/openebs.io_blockdeviceclaims_crd.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: blockdeviceclaims.openebs.io
+spec:
+  group: openebs.io
+  names:
+    kind: BlockDeviceClaim
+    listKind: BlockDeviceClaimList
+    plural: blockdeviceclaims
+    singular: blockdeviceclaim
```

Comment:
shortName need to be added

Reply:
@akhilerm Wasn't able to find the marker for it in kubebuilder doc. 
https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
shortName is a field in the crd spec.

Reply:
Yeah, but CRDs are generated by controller-gen, we shouldn't manually add it, I guess.

Reply:
ooh. ohkay. How is it done in the other openebs CRDs?

Reply:
@harshthakur9030 Can you check [this](https://github.com/openebs/api/blob/master/config/crds/all-crds.yaml). We should add the shortName.
Else `kubectl get bd` command wont work.